legal-api/src/legal_api/config.py

```diff
@@ -144,12 +144,17 @@ class _Config():  # pylint: disable=too-few-public-methods
 
     NAICS_API_URL = os.getenv('NAICS_API_URL', 'https://NAICS_API_URL/api/v2/naics')
 
-    # Digital Credential configuration values
+    # Traction ACA-Py tenant settings to issue credentials from
     TRACTION_API_URL = os.getenv('TRACTION_API_URL')
-    TRACTION_API_TOKEN = os.getenv('TRACTION_API_TOKEN')
+    TRACTION_TENANT_ID = os.getenv('TRACTION_TENANT_ID')
+    TRACTION_API_KEY = os.getenv('TRACTION_API_KEY')
     TRACTION_PUBLIC_SCHEMA_DID = os.getenv('TRACTION_PUBLIC_SCHEMA_DID')
     TRACTION_PUBLIC_ISSUER_DID = os.getenv('TRACTION_PUBLIC_ISSUER_DID')
 
+    # Web socket settings
+    WS_ALLOWED_ORIGINS = os.getenv('WS_ALLOWED_ORIGINS')
```

Comment:
Are you getting Patrick to help you configure this in 1Password?

If not, I can help. Let me know the value.